src/components/Proposal/ProposalHistoryRow.tsx

```diff
@@ -86,13 +89,13 @@ class ProposalHistoryRow extends React.Component<IProps, IState> {
       currentAccountVoteAmount = new BN(currentVote.staticState.amount);
     }
 
-    let currentStake: Stake;
     if (stakesOfCurrentUser.length > 0) {
-      currentStake = stakesOfCurrentUser[0];
-    }
-    if (currentStake) {
-      currentAccountPrediction = currentStake.staticState.outcome;
-      currentAccountStakeAmount = new BN(currentStake.staticState.amount);
+      const totalStakes = stakesOfCurrentUser
+        .map((stake): BN => stake.staticState.amount)
+        .reduce((prev: BN, current: BN)  => { return prev.add(current); });
+      
+      currentAccountPrediction = stakesOfCurrentUser[0].staticState.outcome;
+      currentAccountStakeAmount = totalStakes;
```

Comment:
why not just set this above instead of create a new variable?

Reply:
nice little catch.  Done